applications/serial_lte_modem/doc/AT_commands.rst

```diff
@@ -1,4 +1,4 @@
-.. _SLM_AT_intro:
+.. _SLM_AT_commands:
```

Comment:
Seems we used the old ref target in the documentation.

To fix the doc build, either add back in the the old target (a heading can have more than one target, preferred one should be listed first), or search the documentation for instances of the old target and replace them with the new one.

Reply:
Thanks for the heads-up. I assumed that nothing else could reference this. I forgot about the SLM shell sample.